core/workspace.js

```diff
@@ -266,7 +266,12 @@ Blockly.Workspace.prototype.getBlocksByType = function(type, ordered) {
     }
     blocks.sort(this.sortObjects_);
   }
-  return blocks;
+
+  var filtered = blocks.filter(function(block) {
+    return !block.isInsertionMarker();
+  });
+
+  return filtered;
```

Comment:
```suggestion
  return blocks.filter(function(block) {
    return !block.isInsertionMarker();
  });
```
This can be turned into one return statement =)

Reply:
Thank you for reviewing. I have committed the suggested changes. 
Thank you @BeksOmega

Reply:
My pleasure! Thank you for the PR :D